src/terminal/parser/OutputStateMachineEngine.cpp

```diff
@@ -298,7 +295,7 @@ bool OutputStateMachineEngine::ActionCsiDispatch(const wchar_t wch,
     unsigned int function = 0;
     DispatchTypes::EraseType eraseType = DispatchTypes::EraseType::ToEnd;
     std::vector<DispatchTypes::GraphicsOptions> graphicsOptions;
-    DispatchTypes::AnsiStatusType deviceStatusType = (DispatchTypes::AnsiStatusType)-1; // there is no default status type.
+    DispatchTypes::AnsiStatusType deviceStatusType = static_cast<DispatchTypes::AnsiStatusType>(-1); // there is no default status type.

```

Comment:
is this enum signed?

Reply:
if not, base type may not hold `-1` properly

Reply:
No, it's not. I made it 0 to match since that doesn't do anything anyway.